packages/web/src/common/store/playlist-library/sagas.ts

```diff
@@ -42,9 +41,10 @@ function* watchUpdatePlaylistLibrary() {
 
       yield* put(accountActions.updatePlaylistLibrary(updatedPlaylistLibrary))
 
-      yield* fork(updateProfileAsync, {
-        metadata: { ...account, playlist_library: updatedPlaylistLibrary }
-      })
+      // feature-tan-query TODO: update user playlist_library
```

Comment:
heads up lets not even make this a part of updating user, let's have a useUpdatePlaylistLibary hook that manages everything

Reply:
yes definitely agree!